### ISSUE TITLE
ST_MAKEPOLYGON

### ISSUE THREAD

**Issue by luiztauffer** [eduardogspereira/pypika-gis#3]
Hi @eduardogspereira ! (você é brasileiro?)
Thanks for the great project!

I've been trying to use some of the functions with google bigquery, but it seems that GeomFromGeoJSON for example does not work well for bigquery. I need to query for where (lon, lat) lies inside a polygon, and I was trying the GeomFromGeoJSON to define the polygon. Do you have a development roadmap, or expect to be able to incorporate new functions at any point?
It would be great to see other functions such as ST_MAKEPOLYGON!

**Comment by eduardogspereira** [comment]
Hi @luiztauffer, sou sim ^^

I was checking the [spatial functions available at bigquery](https://cloud.google.com/bigquery/docs/gis-data), they use the GeogFromGeoJSON instead of GeomFromGeoJSON.

I inserted the GeogFromGeoJSON and the others functions spatial functions listed at bigquery and released at pypika-gis v1.3.0. I updated the README with all the functions available :).

Let me know if it works fine for you 🤠
